yellowbrick/features/__init__.py

```diff
@@ -21,7 +21,6 @@
 from .pcoords import ParallelCoordinates, parallel_coordinates
 from .radviz import RadialVisualizer, RadViz, radviz
 from .rankd import Rank1D, rank1d, Rank2D, rank2d
-from .scatter import ScatterViz, ScatterVisualizer, scatterviz
```

Comment:
Hey @bbengfort - it looks like we removed this import line from `yellowbrick/features/__init__.py` but maybe didn't add it to `yellowbrick/contrib/__init__.py`?

Reply:
Hi @rebeccabilbro sorry, I didn't see this until just now. I guess I didn't really intend to hoist the visualizers into the `contrib` package; was envisioning that each `contrib` package would be isolated (e.g. to protect optional dependencies). However, in this case, if you feel like our users would benefit from being able to more directly import the `ScatterVisualizer`, please feel free to add that import statement.

Reply:
Ok, will do!